### DISCUSSION TITLE
WAMP no longer avilable?

### DISCUSSION THREAD

**Discussion by ScottVordem** [ampache/ampache#3617] [General]
## Description
Following the Windows tutorial says to download and install WAMP. Clicking the provided link leads to a page saying that WAMP is discontinued and seemingly no download options,

### To reproduce

1. Go to the installation guide for Windows on the Wiki
2. Click on the link to download WAMP

### Screenshots

![image](https://github.com/ampache/ampache/assets/107659193/67847bf4-369c-4341-8699-c3be402e0c83)
I'm not knowledgeable enough in this stuff to know what to use instead.

<!-- To enable logging, visit https://github.com/ampache/ampache/wiki/Troubleshooting#enable-logging

**Comment by mitchray** [comment]
The WAMP project is https://www.wampserver.com/en/

Its only the Bitnami WAMP package which is discontinued, and being a third party packager thats fine you don't need it.

WAMP can be downloaded here: https://sourceforge.net/projects/wampserver/files/WampServer%203/WampServer%203.0.0/wampserver3.3.0_x64.exe/download

**Comment by lachlan-00** [comment]
There are a lot of options is wampserver preferred? i've used ampps a lot because they package ampache in softaculous https://ampps.com/downloads/
